### PR TITLE
CONN-1419 ConnectAdminGUI weblogic updates for configuration and contain...

### PR DESCRIPTION
**Pull request by jsmith2**
...er service caching

Removed jax.ws from preferred packaging.
Cached the container service.
Logged the admin gui proxy type.

### CODE REVIEW

File: Product/Production/Adapters/General/CONNECTAdminGUI/src/main/java/gov/hhs/fha/nhinc/admingui/proxy/DirectConfigProxyWebServiceUnsecuredContainerImpl.java [hunk 1]
```diff
@@ -333,15 +333,17 @@ public void removeAddress(String addressEmail) throws Exception{
    
     private ConfigurationServiceImplService getConfigService() throws ConnectionManagerException, MalformedURLException{
         String url = oProxyHelper.getAdapterEndPointFromConnectionManager(DirectConfigConstants.DIRECT_CONFIG_SERVICE_NAME);
-
-        ConfigurationServiceImplService cfService = new ConfigurationServiceImplService(new URL(url+"?wsdl"));
+        
+        if(cfService == null) {
+            cfService = new ConfigurationServiceImplService(new URL(url+"?wsdl"));
```
Comment (christophermay07) [inline Product/Production/Adapters/General/CONNECTAdminGUI/src/main/java/gov/hhs/fha/nhinc/admingui/proxy/DirectConfigProxyWebServiceUnsecuredContainerImpl.java]: Should this be initialized in a static / thread-safe context?

**Comment by christophermay07** [comment]
I've reviewed this PR, endorsing for merge without further comment.

**Comment by nareshcgi** [comment]
retest this please

**Comment by nareshcgi** [comment]
Reviewed the PR and looks good.

**Pull request merged**